demo/fatalerror.js

```diff
@@ -1,5 +1,5 @@
-// Example - generation of Nodereport on fatal error (Javascript heap OOM)
-require('nodereport').setEvents("fatalerror");
+// Example - generation of node-report on fatal error (Javascript heap OOM)
```

Comment:
I won't keep mentioning this. A report is a "report" not a "NodeReport" or a "node-report".